package.json

```diff
@@ -16,19 +16,23 @@
     "qdequele <quentin@meilisearch.com>"
   ],
   "license": "MIT",
-  "main": "./dist/bundles/meilisearch.cjs",
-  "module": "./dist/bundles/meilisearch.mjs",
-  "typings": "./dist/types/index.d.ts",
-  "types": "./dist/types/index.d.ts",
```

Comment:
EXPLAINER: There are no longer required, as `exports` below contains the types, and TypeScript primarily looks there.